libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/UrlUtils.java

```diff
@@ -254,4 +254,22 @@ public static String appendUrlParameters(String url, Map<String, String> paramet
         }
         return uriBuilder.build().toString();
     }
+
+    /**
+     * Extracts the subdomain from a domain string.
+     * @param domain A domain is expected. Protocol is optional
+     * @return The subdomain or an empty string. Note that if the domain does not specify a subdomain
+     * (e.g. "http://example.com" then the host is returned.
+     */
+    public static String extractSubDomain(String domain) {
```

Comment:
I should've caught this in my previous review, but I'm concerned about how this returns the host if there isn't a subdomain. If I called a routine named "extractSubDomain" I would expect it to return only the subdomain if there is one, otherwise it returns an empty string.